tests/LitEmojiTest.php

```diff
@@ -44,8 +44,8 @@ public function testHtmlToUnicode()
 
     public function testUnicodeToShortcodeTiming()
     {
-        $text = LitEmoji::encodeShortcode(file_get_contents(__DIR__ . '/UnicodeIpsum'));
-        $this->assertEquals(file_get_contents(__DIR__ . '/ShortcodeIpsum'), $text);
+        $text = LitEmoji::encodeShortcode(file_get_contents(__DIR__.'/UnicodeIpsum'));
```

Comment:
Could you please revert the removal of whitespace around `.`. Perhaps this was an automatic change based on your editor settings, but it's our code style preference that this remains as it was.

Reply:
Definitely a phpcsfixer rule, I did revert those changes